pkg/nordic_softdevice_ble/src/ble-mac.h

```diff
@@ -0,0 +1,50 @@
+#ifndef BLE_MAC_H
+#define BLE_MAC_H
+
+typedef enum {
+    BLE_EVENT_RX_DONE = 20000,
+} ble_mac_event_enum_t;
+
+#define BLE_IFACE_ADDED             (10000)
+
+#define BLE_SIXLOWPAN_MTU           (1280U)
+#define BLE_SIXLOWPAN_L2_ADDR_LEN   (8)
```

Comment:
I was wondering why somewhere it's 6, somewhere it's 8. Well, I guess we should stick with the linux mainline convention for a while?